SwissTransferFeatures/SettingsView/EditSettings/EditSettingsModel.swift

```diff
@@ -0,0 +1,185 @@
+/*
+ Infomaniak SwissTransfer - iOS App
+ Copyright (C) 2024 Infomaniak Network SA
+
+ This program is free software: you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+ This program is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License
+ along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import InfomaniakDI
+import STCore
+import STResources
+import SwiftUI
+import SwissTransferCore
+import SwissTransferCoreUI
+
+/// Something used to populate a `EditSettingsView`
+struct EditCellModel: Hashable, Equatable {
+    let label: String
+    let action: () -> Void
+    let leftIconAsset: STResourcesImages?
+    let rightIconAsset: STResourcesImages?
+
+    init(label: String,
+         action: @escaping () -> Void,
+         leftIconAsset: STResourcesImages? = nil,
+         rightIconAsset: STResourcesImages? = nil) {
+        self.label = label
+        self.action = action
+        self.leftIconAsset = leftIconAsset
+        self.rightIconAsset = rightIconAsset
+    }
+
+    static func == (lhs: Self, rhs: Self) -> Bool {
+        lhs.label == rhs.label
+    }
+
+    func hash(into hasher: inout Hasher) {
+        hasher.combine(label)
+    }
+}
+
+protocol EditSettingsModel {
```

Comment:
Discussed with @valentinperignon 
It is probably cleaner to decorate `STCore` types than have wrapping `Model` Types for each of them. It is something to posibly explore.